upup/pkg/fi/cloudup/phase.go

```diff
@@ -39,3 +39,11 @@ var Phases = sets.NewString(
 	string(PhaseNetwork),
 	string(PhaseCluster),
 )
+
+// PhaseLifecycleMap map of phases to lifecycle name
+var PhaseLifecycleMap = map[Phase]string{
+	PhaseStageAssets: "stageAssetsLifecycle",
```

Comment:
I don't think the word "lifecycle" belongs in here - these are phases, not lifecycles